src/resources/dams42json-export.xsl

```diff
@@ -481,18 +481,6 @@
                        <xsl:with-param name="val"><xsl:value-of select="."/></xsl:with-param>
                     </xsl:call-template>
                 </xsl:when>
-                <xsl:when test="local-name() = 'permission'">
-                    <xsl:call-template name="appendJsonObject">
-                       <xsl:with-param name="key">otherRights:permission/type</xsl:with-param>
-                       <xsl:with-param name="val"><xsl:value-of select="dams:Permission/dams:type"/></xsl:with-param>
```

Comment:
why were these permission/restriction parts removed?

Reply:
nevermind! i found the comment here https://github.com/ucsdlib/damsmanager/issues/301#issuecomment-480953535 explaining why that was done.